Test/Unittest/test_domain/test_collections.py

```diff
@@ -1,25 +1,29 @@
-from math import inf
 from random import choice, shuffle
+from sys import float_info
 from unittest import TestCase
 
 from Test.Unittest.test_helpers.anons import anon_float, anon_range
 from domain.collections import Range
 
 
+FLOAT_MAX_VALUE = float_info.max
+FLOAT_MIN_VALUE = -1 * FLOAT_MAX_VALUE
```

Comment:
These are likely unused and if so should be deleted

Reply:
The generic Range class still supports infinite values as those are valid floats. These are needed for those tests